clients/drcachesim/tracer/raw2trace.cpp

```diff
@@ -570,6 +570,17 @@ raw2trace_t::process_offline_entry(raw2trace_thread_data_t *tdata,
                 buf, (trace_marker_type_t)in_entry->extended.valueB, marker_val);
             if (in_entry->extended.valueB == TRACE_MARKER_TYPE_KERNEL_EVENT) {
                 log(4, "Signal/exception between bbs\n");
+                if (tdata->rseq_past_end_) {
+                    err = adjust_and_emit_rseq_buffer(tdata, marker_val);
+                    if (!err.empty())
+                        return err;
+                }
+            } else if (in_entry->extended.valueB == TRACE_MARKER_TYPE_RSEQ_ENTRY) {
+                log(4, "--- Reached rseq entry (end=0x%zx): buffering all output ---\n",
```

Comment:
Hmm, I added the dashes to help visually identify the rseq regions in the verbose output and found it very useful.  Unless you feel strongly I would prefer to keep them.

Reply:
>  found it very useful

Was that just because the rseq ones were more relevant for the current work? It may not be that case for a future user.

Keeping them would mean we're treating rseq logs as special. I guess that's ok.

Reply:
> Was that just because the rseq ones were more relevant for the current work? It may not be that case for a future user.

Partly, but given the very different behavior around rseq in general (buffering and rolling back) I think highlighting it for any other raw2trace log focus will still be relevant.